### ISSUE TITLE
Material Sheet crate manifests aren't accurately reflecting the new mat values

### ISSUE THREAD

**Issue by Gristlebee** [tgstation/tgstation#76802]
## Reproduction:

![image](https://github.com/tgstation/tgstation/assets/56049844/01f5f52c-102c-4d48-9f4f-1f5a80775fa3)

Order a crate of 50 sheet mat crate. (Glass/Metal/Plasteel etc)
Check the crate manifest and see the discrepancy, since 50 sheets of a mat in now considered "one".

**Comment by TheVekter** [comment]
I don't think this is a mat value issue, it's got to do with #76012 changing it so instances of items show up grouped on a manifest instead of line by line. Since a stack of iron just shows up as a single "iron" in your hand, that's how it shows here.
